docs/contracts/function-modifiers.rst

```diff
@@ -39,7 +39,7 @@ if they are marked ``virtual``. For details, please see
         }
     }
 
-    contract mortal is owned {
```

Comment:
This should have resulted in a test failure, because the contract is referenced below without change.

Reply:
Missing test runs usually result from wrong code indentation and should have been fixed by https://github.com/ethereum/solidity/pull/7564. Perhaps we should check it again.

Reply:
Created an issue about it.

Reply:
it is? I can't find `mortal` anywhere in the whole repo?

Reply:
You are right, it was weirdly inheriting from `owned`...: https://github.com/ethereum/solidity/pull/8111/files#diff-2e640333fd552416d91547f68fb74857L61